hack/remove-members.go

```diff
@@ -0,0 +1,184 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var dryrun bool
+var repoRoot string
+
+func main() {
+	flag.StringVar(&repoRoot, "root", ".", "Root of the repository")
+	flag.BoolVar(&dryrun, "dryrun", true, "Enable Dryrun or not")
+
+	flag.Parse()
+
+	if len(flag.Args()) < 1 {
+		fmt.Print("Usage: remove-members [flags] <file-containing-members-list>\n\nFlags:\n\t-root\tstring\tpath to root directory of the repository\n\t-dryrun\tboolean\tEnable/Disable dryrun (Default: Enabled)")
+		os.Exit(0)
+	}
+
+	memberList, err := readMemberList(flag.Args()[0])
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	configPath := repoRoot + "/config"
+
+	err = removeMembers(memberList, configPath)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+}
+
+//readMemberList reads the list of members to be removed from the given filepath
+func readMemberList(path string) ([]string, error) {
+	file, err := os.Open(path)
```

Comment:
@mrbobbytables is the inactive member list a yaml file? how is that generated?

Reply:
It's been a text file with 1 member per line. The list is generated from this sheet after some manual vetting: https://docs.google.com/spreadsheets/d/1jqxMOo9f1EG72i4sGE7g6RlNPvQ4_qfOx9K-UM4YsLw/edit

Reply:
Thanks! One more follow up question - how does one generate this sheet?

Want to remove you as a single point of failure :)

Reply:
The big stuff to regen it are in the notes tab in the sheet 👍 I thought it was owned/under the sig contribex shared dir, but it looks like I'm the only owner atm. I'll move it over in a bit.